internal/core/input.rs

```diff
@@ -276,8 +276,16 @@ impl KeyEvent {
         };
 
         match keycode {
-            key_codes::Backspace => Some(TextShortcut::DeleteBackward),
-            key_codes::Delete => Some(TextShortcut::DeleteForward),
+            key_codes::Backspace => Some(if move_mod {
```

Comment:
The existing code does is ordered by modifier and looks into the key itself second.
this code looks into the key first and then checks the modifier.
perhaps move that to the move_mod if block?

Reply:
Good idea, done in 49ea24c44a6ff2b00f007c5165d479bc3a28ddca